cmake/external/mkldnn.cmake

```diff
@@ -71,13 +71,16 @@ ExternalProject_Add(
                         -DMKLROOT=${MKLML_ROOT}
                         -DCMAKE_C_FLAGS=${MKLDNN_CFLAG}
                         -DCMAKE_CXX_FLAGS=${MKLDNN_CXXFLAG}
-                        -DMKLDNN_BUILD_TESTS=OFF -DMKLDNN_BUILD_EXAMPLES=OFF
+                        -DDNNL_BUILD_TESTS=OFF -DDNNL_BUILD_EXAMPLES=OFF
     CMAKE_CACHE_ARGS    -DCMAKE_INSTALL_PREFIX:PATH=${MKLDNN_INSTALL_DIR}
 )
 if(WIN32)
-    SET(MKLDNN_LIB "${MKLDNN_INSTALL_DIR}/${LIBDIR}/mkldnn.lib" CACHE FILEPATH "mkldnn library." FORCE)
+    SET(MKLDNN_LIB "${MKLDNN_INSTALL_DIR}/${LIBDIR}/dnnl.lib" CACHE FILEPATH "mkldnn library." FORCE)
+    SET(MKLDNN_LIB_OLD "${MKLDNN_INSTALL_DIR}/${LIBDIR}/mkldnn.lib" CACHE FILEPATH "mkldnn library." FORCE)
```

Comment:
dnnl.lib is the import library of dnnl.dll, copy dnnl.lib to mkldnn.lib, mkldnn.lib will still import dnnl.dll, so mkldnn.dll will not work. So can you generate mkldnn.dll directly? or, use custom command in cmake to generate mkldnn.lib from mkldnn.dll,rather than copy mkldnn.lib from dnnl.lib.

Reply:
I don't understand why mkldnn.lib importing dnnl.dll is a problem if we will have all 4 libraries in a folder?